docs/source/run/parameters.rst

```diff
@@ -621,43 +617,67 @@ Parameters starting with ``lasers.`` apply to all laser pulses, parameters start
 Diagnostic parameters
 ---------------------
 
+* ``diagnostic.output_period`` (`integer`) optional (default `-1`)
+    Output period for all diagnostics. Field or beam specific diagnostics can overwrite this parameter.
+    No output is given for ``diagnostic.output_period = -1``.
+    **Warning:** ``diagnostic.output_period = 0`` will make the simulation crash.
```

Comment:
Should we remove this warning and change the code so `if (output_period == 0) output_period = -1;`? I guess that if a user sets this to 0, they probably don't want diagnostics. That applied to the overwrites below.
```suggestion
    **Warning:** ``diagnostic.output_period = 0`` will make the simulation crash.
```